docker-dynamodb/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  db:
+    image: constellation-db
+    deploy:
+      replicas: 1
+      resources:
+        cpus: 0.1
+        memory: 100M
+    volumes:
+      - ./data:/opt/dynamodb/data
```

Comment:
I guess that means part of the build process (maybe?) should be to copy the repo's datastore file to /opt/dyanmodb/data